wandb/sdk/launch/agent/agent.py

```diff
@@ -56,10 +56,11 @@ def __init__(self, api: Api, config: Dict[str, Any]):
         self._cwd = os.getcwd()
         self._namespace = wandb.util.generate_id()
         self._access = _convert_access("project")
-        if config.get("max_jobs") == -1:
+        max_jobs_from_config = int(config.get("max_jobs", 1))
```

Comment:
this case is probably unnecessary now. But I propose leaving it until we've fully typed the agent config to prevent any future issues